### PR TITLE
[zh-cn] update delete-stateful-set.md

### PR DESCRIPTION
**Pull request by Zhuzhenghao**
Add relevant glossary tooltips.

Guide reader to learn more about StatefulSet.

Sync with `content/en/docs/tasks/run-application/delete-stateful-set.md`

### CODE REVIEW

File: content/zh-cn/docs/tasks/run-application/delete-stateful-set.md [hunk 1]
```diff
@@ -133,14 +133,14 @@ In the example above, the Pods have the label `app.kubernetes.io/name=MyApp`; su
...
-If you find that some pods in your StatefulSet are stuck in the 'Terminating' or 'Unknown' states for an extended period of time, you may need to manually intervene to forcefully delete the pods from the apiserver. This is a potentially dangerous task. Refer to [Deleting StatefulSet Pods](/docs/tasks/manage-stateful-set/delete-pods/) for details.
+If you find that some pods in your StatefulSet are stuck in the 'Terminating' or 'Unknown' states for an extended period of time, you may need to manually intervene to forcefully delete the pods from the apiserver. This is a potentially dangerous task. Refer to [Force Delete StatefulSet Pods](/docs/tasks/run-application/force-delete-stateful-set-pod/) for details.
 -->
 ### 强制删除 StatefulSet 的 Pod
 
 如果你发现 StatefulSet 的某些 Pod 长时间处于 'Terminating' 或者 'Unknown' 状态，
 则可能需要手动干预以强制从 API 服务器中删除这些 Pod。
 这是一项有点危险的任务。详细信息请阅读
-[删除 StatefulSet 类型的 Pods](/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod/)。
+[强制删除 StatefulSet 的 Pods](/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod/)。
```
Comment (howieyuen) [inline content/zh-cn/docs/tasks/run-application/delete-stateful-set.md:143]:
```suggestion
[强制删除 StatefulSet 的 Pod](/zh-cn/docs/tasks/run-application/force-delete-stateful-set-pod/)。
```

**Review by howieyuen — COMMENTED** [review]
可以检查一下全文中 Pod 作为 keyword 的出现时，首字母大写且保持单数形式

File: content/zh-cn/docs/tasks/run-application/delete-stateful-set.md [hunk 2]
```diff
@@ -111,9 +111,9 @@ Use caution when deleting a PVC, as it may lead to data loss.
 <!--
 ### Complete deletion of a StatefulSet
 
-To simply delete everything in a StatefulSet, including the associated pods, you can run a series of commands similar to the following:
+To delete everything in a StatefulSet, including the associated pods, you can run a series of commands similar to the following:
 -->
-### 完全删除 StatefulSet  {#complete-deletion-of-a-statefulset}
+### 完全删除 StatefulSet {#complete-deletion-of-a-statefulset}
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/run-application/delete-stateful-set.md:116]: Changes like this are unnecessary.
Whitespaces are for readability. Unifying this into a single space doesn't add much value.

File: content/zh-cn/docs/tasks/run-application/delete-stateful-set.md (hunk 1 above)
Comment (Zhuzhenghao) [reply content/zh-cn/docs/tasks/run-application/delete-stateful-set.md]: 发现了两处：
https://github.com/kubernetes/website/pull/38752/files#diff-932b3ab5dcea0544bbf3f58a643eb0a88924be85aaeaeed22cfc0d6a9dc1d30bR117

https://github.com/kubernetes/website/pull/38752/files#diff-932b3ab5dcea0544bbf3f58a643eb0a88924be85aaeaeed22cfc0d6a9dc1d30bR149

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**